weblab/accounts/views.py

```diff
@@ -54,13 +54,17 @@ def get_context_data(self, **kwargs):
         return super().get_context_data(**kwargs)
 
 
-class UserDeleteView(LoginRequiredMixin, DeleteView):
-    """
-    Delete a user
-    """
-    template_name = 'registration/delete_user.html'
+class UserDeleteView(UserPassesTestMixin, DeleteView):
+    template_name = 'registration/account_confirm_delete.html'
     model = User
 
+    """
```

Comment:
The docstring should be the first thing within the class, before `template_name` etc.